src/marker-layer.coffee

```diff
@@ -115,6 +115,12 @@ class MarkerLayer
           markerIds = filterSet(markerIds, @index.findStartingAt(Point.fromObject(value)))
         when 'endPosition'
           markerIds = filterSet(markerIds, @index.findEndingAt(Point.fromObject(value)))
+        when 'startsInRange'
```

Comment:
I purposefully avoided documenting `startsInRange` and `endsInRange` because they are only being used internally but I can definitely see exposing them at some point.

What do you think @maxbrunsfeld and @nathansobo? Should we wait a little more or should we go for it? 💭

Reply:
I think you might as well document it. It's pretty low risk.